### PR TITLE
[Merged by Bors] - Log fixes

### PR DESCRIPTION
**Pull request by gavraz**
## Motivation
Improve log messages.

## Changes
More informatic logs.

### CODE REVIEW

File: activation/activation.go [hunk 1]
```diff
@@ -371,7 +371,7 @@ func (b *Builder) PublishActivationTx() error {
 		return fmt.Errorf("failed to build nipst: %v", err)
 	}
 
-	b.log.With().Info("awaiting publication epoch",
+	b.log.With().Info("awaiting publication epoch of atx",
```
Comment (noamnelke) [inline activation/activation.go]:
```suggestion
	b.log.With().Info("awaiting atx publication epoch",
```

File: miner/block_builder.go [hunk 2]
```diff
@@ -365,7 +372,23 @@ func (t *BlockBuilder) handleGossipAtx(data service.GossipMessage) {
...
 	atx.CalcAndSetId()
 
-	t.With().Info("got new ATX", log.AtxId(atx.ShortString()), log.LayerId(uint64(atx.PubLayerIdx)))
+	commitmentStr := "nil"
+	if atx.Commitment != nil {
+		commitmentStr = atx.Commitment.String()
+	}
+
+	t.With().Info("got new ATX",
+		log.String("sender_id", atx.NodeId.ShortString()),
```
Comment (noamnelke) [inline miner/block_builder.go:381]: in `ATX published` this is called `miner` - maybe we should be consistent, one way or another. cc: @barakshani

File: miner/block_builder.go [hunk 3]
```diff
@@ -365,7 +372,23 @@ func (t *BlockBuilder) handleGossipAtx(data service.GossipMessage) {
...
+		log.AtxId(atx.ShortString()),
+		log.String("prev_atx_id", atx.PrevATXId.ShortString()),
+		log.String("post_atx_id", atx.PositioningAtx.ShortString()),
```
Comment (noamnelke) [inline miner/block_builder.go]:
```suggestion
		log.String("pos_atx_id", atx.PositioningAtx.ShortString()),
```
Should also fix the same field in `ATX Published`

File: sync/syncer.go [hunk 4]
```diff
@@ -520,15 +520,18 @@ func (s *Syncer) validateBlockView(blk *types.Block) bool {
 	ch := make(chan bool, 1)
 	defer close(ch)
 	foo := func(res bool) error {
-		s.Info("validate view for %s done %v", blk.Id(), res)
+		s.With().Info("validate view",
```
Comment (noamnelke) [inline sync/syncer.go]:
```suggestion
		s.With().Info("view validated",
```

File: activation/activation.go [hunk 5]
```diff
@@ -441,6 +446,10 @@ func (b *Builder) PublishActivationTx() error {
 		log.Uint32("active_set", atx.ActiveSetSize),
 		log.String("miner", b.nodeId.Key[:5]),
```
Comment (noamnelke) [inline activation/activation.go]:
```suggestion
		log.String("miner", b.nodeId.ShortString()),
```

**Review by noamnelke — CHANGES_REQUESTED** [review]

File: miner/block_builder.go (hunk 2 above)
Comment (gavraz) [reply miner/block_builder.go:381]: No need

**Review by noamnelke — APPROVED** [review]

**Comment by gavraz** [comment]
bors merge

**Pull request closed without merging**